packages/addons-inject/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@readthedocs/addons-inject",
```

Comment:
we should take over this namespace if we haven't already :)

Reply:
Yeah might as well, we use this scope on all our packages.

Reply:
I started this, someone already claimed `readthedocs` organization. I reached out to support on this.